core/src/main/java/io/questdb/std/datetime/millitime/GenericDateFormat.java

```diff
@@ -522,9 +535,12 @@ public long parse(CharSequence in, int lo, int hi, DateLocale locale) throws Num
                     pos += Numbers.decodeHighInt(l);
                     break;
 
```

Comment:
Duplicates case blocks ?

Reply:
Not sure I can follow here, cases are already grouped, or am I missing something?

Reply:
Sorry, my mistake. GH made it look like they were part of same switch statement .